src/CosObjectHelpers.jl

```diff
@@ -8,6 +8,7 @@ import Rectangle: Rect
 function convert(::Type{CDTextString}, xstr::CosXString)
     feff = [LATIN_F, LATIN_E, LATIN_F, LATIN_F]
     FEFF = [LATIN_UPPER_F, LATIN_UPPER_E, LATIN_UPPER_F, LATIN_UPPER_F]
+    length(xstr.val) < 4 && return CDTextString("")
```

Comment:
PDF Spec Section 7.9.2:

> PDF supports one fundamental string object (see 7.3.4, "String Objects"). The string object shall be further
qualified as a text string, a PDFDocEncoded string, ASCII string, or byte string. The further qualification reflects
the encoding used to represent the characters or glyphs described by the string.

Only `CosXStrings` which have a valid UTF2BE specifier qualifies as a `CDTextString`. While others could technically be `byte string` this has not been implemented yet. We may create a `CDBinaryString` later to address those needs. In my understanding only `CDBinaryString` should represent a null `CosXString`. 

Hence, I am not quite convinced, this change may be actually needed. If you are interested in implementing a `CDBinaryString` then you can modify this PR in that direction.

Reply:
In my file the property that made the crash has value `<>` (empty hex string).
I have defined binary string as follows:
```julia
const CDBinaryString = Vector{UInt8}
```
and then changed code in `pdDocGetInfo` like this:
```julia
        for (key, val) in get(obj)
            skey = CDTextString(key)
            if val isa CosXString && length(val.val) < 4
                @warn "Unable to decode $skey"
                dInfo[skey] = CDBinaryString(val)
            else
                dInfo[skey] = (skey == "CreationDate") || (skey == "ModDate") ?
                              CDDate(val) : CDTextString(val)
            end
        end
```
And I got a valid info object (what is okay for me):
```
Dict{String,Union{CDDate, Array{UInt8,1}, String}} with 8 entries:
  "Producer"     => "Scribus PDF Library 1.3.3.13"
  "Author"       => UInt8[]
  ....
```
Would this change be okay for now or you mean any more sophisticated implementation?
Is the warning necessary in your opinion?

Reply:
It may be far simpler to keep the function to below. No changes to any internal code may be needed nor any `CDBinaryString` etc. When a value is invalid, it should be a `CosNull` than an  array of null bytes. 

```
diff --git a/src/PDDoc.jl b/src/PDDoc.jl
index cf58165..45b4205 100644
--- a/src/PDDoc.jl
+++ b/src/PDDoc.jl
@@ -129,11 +129,16 @@ Please refer to the PDF specification for further details.
 """
 function pdDocGetInfo(doc::PDDoc)
     obj = cosDocGetInfo(doc.cosDoc)
-    dInfo = Dict{CDTextString, Union{CDTextString, CDDate}}()
+    dInfo = Dict{CDTextString, Union{CDTextString, CDDate, CosObject}}()
     for (key, val) in get(obj)
         skey = CDTextString(key)
-        dInfo[skey] = (skey == "CreationDate") || (skey == "ModDate") ?
-                      CDDate(val) : CDTextString(val)
+        retval = CosNull
+        try
+            retval = (skey == "CreationDate") || (skey == "ModDate") ?
+                     CDDate(val) : CDTextString(val)
+        catch
+        end
+        dInfo[skey] = retval
     end
     return dInfo
 end
```

Output: 
```
julia> pdDocGetInfo(doc)
Dict{String,Union{CDDate, String, CosObject}} with 8 entries:
  "Producer"     => "Scribus PDF Library 1.3.3.13"
  "Author"       => null
  "CreationDate" => 2009-08-07T19:26:00 UTC
  "Keywords"     => null
  "ModDate"      => 2018-10-29T11:30:00 UTC
  "Creator"      => "Scribus 1.3.3.13"
  "Trapped"      => /False
  "Title"        => null
```

Please add a test case with the file you have attached in the bug.